projects/cashmere/src/lib/pop/directives/popover-anchor.directive.ts

```diff
@@ -104,6 +106,15 @@ export class HcPopoverAnchorDirective implements OnInit, AfterContentInit, OnDes
         this._attachedPopover.maxWidth = val;
     }
 
+    /** Whether the popover should return focus to the previously focused element after closing. *Defaults to `true`.* */
+    @Input()
+    get restoreFocus() {
```

Comment:
Sorry Hayden - I should have caught this on the first review.  Can you add `: boolean` to your restoreFocus getter.  It'll just make the documentation look better if there is a type associated with it.